text/0000-builder-spec.md

```diff
@@ -0,0 +1,295 @@
+# Meta
+[meta]: #meta
+- Name: Builder Spec
+- Start Date: 2020-09-11
+- Author(s): [@dfreilich](https://github.com/dfreilich)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+A Builder is a critical tool in the CNB ecosystem, and yet, is largely [unspecified][spec-issue-builder]. This RFC proposes that we should add a rigorous versioned specification of builders as an extension to the core specification, and clarify the existence of builders in the:
+* [distribution specification][distribution-spec]
```

Comment:
I think that since we use builders in a lot of contexts, to distribute things, it would be weird for us not to at least acknowledge their existence in the distribution spec, even if we don't require them. Similar to the addition in the platform spec that builders are fully functioning stack build image, say that builders are similar artifacts to buildpackages, and MAY be used by some platforms.